drltrace/scripts/gen_drlttrace_config.py

```diff
@@ -0,0 +1,379 @@
+"LPOSVERSIONINFOEXW":"OSVERSIONINFOEXW*", "LPOSVERSIONINFOEXA":"OSVERSIONINFOEXA*",
+"LPOSVERSIONINFOA":"OSVERSIONINFOA*", "LPNLSVERSIONINFOEX":"NLSVERSIONINFOEX*",
+"LPMEMORYSTATUSEX":"MEMORYSTATUSEX*", "LPMEMORYSTATUS":"MEMORYSTATUS*", "LPLONG":"LONG*",
+"LPLDT_ENTRY":"LDT_ENTRY*", "LPHANDLE":"HANDLE*", "LPFILE_ID_DESCRIPTOR":"FILE_ID_DESCRIPTOR*",
+"LPDEBUG_EVENT":"DEBUG_EVENT*", "LPCPINFOEXW":"CPINFOEXW*", "LPCPINFOEXA":"CPINFOEXA*",
+"LPCPINFO":"CPINFO*", "LPCONTEXT":"CONTEXT*", "LPCOMMPROP":"COMMPROP*",
+"LPBY_HANDLE_FILE_INFORMATION":"BY_HANDLE_FILE_INFORMATION*", "SYSTEMTIME":"SYSTEMTIME",
+"REGSAM":"REGSAM", "HFILE":"HFILE", "USHORT":"USHORT", "UINT_PTR":"UINT_PTR", "SMALL_RECT":"SMALL_RECT",
+"UCHAR":"uchar", "LONGLONG":"LONGLONG", "GUID":"GUID", "GET_FILEEX_INFO_LEVELS":"GET_FILEEX_INFO_LEVELS",
+"GEOID":"GEOID", "FINDEX_SEARCH_OPS":"FINDEX_SEARCH_OPS", "FINDEX_INFO_LEVELS":"FINDEX_INFO_LEVELS",
+"COMPUTER_NAME_FORMAT":"COMPUTER_NAME_FORMAT", "CHAR_INFO":"CHAR_INFO", "ULONGLONG":"ULONGLONG",
+"NLS_FUNCTION":"NLS_FUNCTION", "DWORD_PTR":"DWORD_PTR", "STREAM_INFO_LEVELS":"STREAM_INFO_LEVELS",
+"SECURITY_INFORMATION":"SECURITY_INFORMATION", "NORM_FORM":"NORM_FORM",
+"JOBOBJECTINFOCLASS":"JOBOBJECTINFOCLASS", "INPUT_RECORD":"INPUT_RECORD",
+"HEAP_INFORMATION_CLASS":"HEAP_INFORMATION_CLASS", "GEOTYPE":"GEOTYPE", "GEOCLASS":"GEOCLASS",
+"FILE_SEGMENT_ELEMENT":"FILE_SEGMENT_ELEMENT", "FILE_INFO_BY_HANDLE_CLASS":"FILE_INFO_BY_HANDLE_CLASS",
+"DWORDLONG":"DWORDLONG","EXCEPTION_POINTERS":"EXCEPTION_POINTERS", "_EXCEPTION_POINTERS":"_EXCEPTION_POINTERS",
+"WOW64_CONTEXT":"WOW64_CONTEXT", "UILANGUAGE_ENUMPROCW":"UILANGUAGE_ENUMPROCW",
+"UILANGUAGE_ENUMPROCA":"UILANGUAGE_ENUMPROCA", "TIMEFMT_ENUMPROCW":"TIMEFMT_ENUMPROCW",
+"TIMEFMT_ENUMPROCEX":"TIMEFMT_ENUMPROCEX", "TIMEFMT_ENUMPROCA":"TIMEFMT_ENUMPROCA",
+"NUMBERFMTW":"NUMBERFMTW",  "LOGICAL_PROCESSOR_RELATIONSHIP":"LOGICAL_PROCESSOR_RELATIONSHIP",
+"LOCALE_ENUMPROCW":"LOCALE_ENUMPROCW", "LOCALE_ENUMPROCEX":"LOCALE_ENUMPROCEX",
+"LOCALE_ENUMPROCA":"LOCALE_ENUMPROCA", "LATENCY_TIME":"LATENCY_TIME", "LANGUAGEGROUP_ENUMPROCW":"LANGUAGEGROUP_ENUMPROCW",
+"LANGUAGEGROUP_ENUMPROCA":"LANGUAGEGROUP_ENUMPROCA", "GROUP_AFFINITY":"GROUP_AFFINITY", "GEO_ENUMPROC":"GEO_ENUMPROC",
+"EXECUTION_STATE":"EXECUTION_STATE", "DYNAMIC_TIME_ZONE_INFORMATION":"DYNAMIC_TIME_ZONE_INFORMATION",
+"CURRENCYFMTW":"CURRENCYFMTW", "CONTEXT":"CONTEXT", "CONSOLE_CURSOR_INFO":"CONSOLE_CURSOR_INFO", "CHAR":"char",
+"APPLICATION_RECOVERY_CALLBACK":"APPLICATION_RECOVERY_CALLBACK", "HRESULT": "HRESULT", "PCNZCH": "char*",
+"LPPROCESS_INFORMATION": "PROCESS_INFORMATION*", "LPINT": "int*", "LSTATUS":"LSTATUS",
+"DEP_SYSTEM_POLICY_TYPE":"DEP_SYSTEM_POLICY_TYPE", "CURRENCYFMTA":"CURRENCYFMTA", "va_list": "va_list"}
+
+
+tokens_ignore_prefix = {"WINAPI", "WINBASEAPI", "APIENTRY", "virtual", "DECLSPEC_NORETURN",
+                        "WINNORMALIZEAPI"}
+
+PATH = "results/" # path where the output of headers_parser.py was saved
```

Comment:
I used caps just to highlight variables that a user would want to change (e.g. results folder).  Anyway, I changed this. Now argparse is used to ask a user for such arguments.